imagestores/s3/store.go

```diff
@@ -0,0 +1,34 @@
+package s3
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/google/uuid"
+)
+
+// StoreImage stores an image in aws s3 and returns the link to it
+func (imageStore *ImageStore) StoreImage(image []byte) (string, error) {
+	uploadInput := imageStore.buildUploadInput(image)
+	result, err := imageStore.uploader.Upload(uploadInput)
+	if err != nil {
+		return "", fmt.Errorf("error while uploading to s3: %s", err)
+	}
+
+	return result.Location, nil
+}
+
+func (imageStore *ImageStore) buildUploadInput(image []byte) *s3manager.UploadInput {
+	key := "moira-plots/" + uuid.New().String()
```

Comment:
Can we use here "github.com/gofrs/uuid" package like in other Moira code? It will be a nice reason to remove the additional dependency.